jobs/build/advisories/Jenkinsfile

```diff
@@ -135,7 +175,7 @@ node {
                         draft << "- ${it.key}: https://errata.devel.redhat.com/advisory/${it.value}"
                     }
                     body = draft.join('\n')
-                    if (params.DRY_RUN) {
+                    if (params.DRY_RUN || params.TEXT_ONLY) {
```

Comment:
I think we need live IDs for text-only advisories (until we get the permission to assign live IDs by ourselves).